BAZProject/TrendingModule/MainView/TrendingTableViewCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  TrendingTableViewCell.swift
+//  BAZProject
+//
+//  Created by avirgilio on 30/01/23.
+//
+
+import UIKit
+
+class TrendingTableViewCell: UITableViewCell {
+    
+    let manageImgs = LoadRemotedata()
+    
+    @IBOutlet weak var imgImageMovie: UIImageView!
+    
+    @IBOutlet weak var lblMovieTitle: UILabel!{
```

Comment:
Cuida los espacios por los posibles warnings de Swiftlint.
y el mismo comentario de que los nombres deben ser totalmente legibles.
```suggestion
    @IBOutlet weak var labelMovieTitle: UILabel! {
```